source/2019-04-19-the-ember-times-issue-95.md

```diff
@@ -114,7 +114,7 @@ responsive: true
 
 ---
 
-## [#embertimes 📰](https://emberjs.com/blog/tags/newsletter.html) 
+## [#embertimes 📰](https://blog.emberjs.com/tags/newsletter.html) 
```

Comment:
oh no! ugh

Thanks for catching this, I made a PR to fix this in the template: https://github.com/ember-learn/ember-blog/pull/91